core-bundle/src/Resources/contao/dca/tl_member.php

```diff
@@ -384,6 +384,11 @@
 		(
 			'eval'                    => array('isBoolean'=>true, 'doNotCopy'=>true, 'tl_class'=>'w50 m12'),
 			'sql'                     => "char(1) NOT NULL default ''"
+		),
+		'backupCodes' => array
+		(
+			'eval'                    => array('doNotCopy'=>true),
+			'sql'                     => "text NULL"
```

Comment:
Can't we use the doctrine array notation here and use a proper JSON column? It should be automatically mapped to text for the platforms that do not support JSON columns.

Reply:
But that's not a dealbreaker as we're not querying on the column anyway.